src/main/java/PageObjects/Railway/ChangePasswordPage.java

```diff
@@ -25,8 +28,22 @@ private WebElement getTxtConfirmPassword(){
         return Constant.WEBDRIVER.findElement(txtConfirmPassword);
     }
     private WebElement getBtnChangePassword(){
+
         return Constant.WEBDRIVER.findElement(btnChangePassword);
     }
+    public WebElement getTleChangePasswordPage(){
+
+        return Constant.WEBDRIVER.findElement(tleChangePasswordPage);
+    }
 
     // Methods
+    public void changePassword(String currentPassword, String newPassword, String confirmPassword){
+        Utilities.scrollToFindElement();
```

Comment:
Method name use to describe the function of that method. This method function is scroll. So please name it as `scrollElement` and that is enough